### PR TITLE
feat(properties): generalize the use and them on loads

### PR DESCRIPTION
**Pull request by TheMaskedTurtle**

### CODE REVIEW

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: ok, but 1 header missing
tests: no back-end issue

File: src/main/java/org/gridsuite/network/map/dto/ElementFormInfosWithProperties.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package org.gridsuite.network.map.dto;
```
[the remark this answers is not in the document]
Comment (TheMaskedTurtle) [reply src/main/java/org/gridsuite/network/map/dto/ElementFormInfosWithProperties.java:7]: Done

[the remark this answers is not in the document]
Comment (Mathieu-Deharbe) [reply src/main/java/org/gridsuite/network/map/dto/ElementFormInfosWithProperties.java:7]: We are in 2024!

**Review by Mathieu-Deharbe — APPROVED** [review]

File: src/main/java/org/gridsuite/network/map/dto/ElementFormInfosWithProperties.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (TheMaskedTurtle) [reply src/main/java/org/gridsuite/network/map/dto/ElementFormInfosWithProperties.java:7]: I forget about it already ! Fixed

**Comment by TheMaskedTurtle** [comment]
> By the way I know this is way too late for this but I don't like the fact that those are just called "properties". That is very imprecise, everything belonging to the load is a property, like p0, q0... I liked the freeProperty denomination that I saw somewhere, because the user can freely add whatever he wants.

Hummm that is not really way too late actually ! We won't change the fact that in PowSyBl it's called properties as opposed to attributes (like p0, q0, etc) but I can still change it here though. Now it's factorized here and in the front so there are not so many places to update it, if you think it's worth it.

**Comment by Mathieu-Deharbe** [comment]
> Hummm that is not really way too late actually ! We won't change the fact that in PowSyBl it's called properties as opposed to attributes (like p0, q0, etc) but I can still change it here though. Now it's factorized here and in the front so there are not so many places to update it, if you think it's worth it.

No I don't think this is worth it. It is better to follow powsybl naming. And it is already quite good that "attributes" and "properties" are differentiated. Thanks for the clarification.

**Review by dbraquart — APPROVED** [review]

**Pull request merged**